tests/yaml_tests/test_yaml.py

```diff
@@ -0,0 +1,47 @@
+import glob
+import os
+
+from eth_utils import (
+    to_tuple,
+)
+
+from ruamel.yaml import (
+    YAML,
+)
+from yaml_test_execution import (
+    execute_test_case,
+)
+
+YAML_BASE_DIR = os.path.abspath(os.path.join(__file__, "../../eth2.0-tests/ssz"))
+YAML_FILES = glob.glob(os.path.join(YAML_BASE_DIR, "**/*.yaml"), recursive=True)
+
+
+@to_tuple
+def load_test_cases(filenames):
+    yaml = YAML()
+
+    for filename in filenames:
+        with open(filename) as f:
+            test = yaml.load(f)
+
+        for index, test_case in enumerate(test["test_cases"]):
+            yield test_case, make_test_id(filename, index)
```

Comment:
One of the wonderful thing about `ruamel.yaml` is that it shows line numbers 🤯. So it would be interesting if path+line is included somewhere, and failing tests can be pinned down to the exact place in the thousand lines of yaml code in a snap.

As an example I hack the test id to show the path+line, and vscode allows to cmd+click to teleport me to the failing test.
```python
        for index, test_case in enumerate(test["test_cases"]):
            path = f"{index} {filename}:{test_case.lc.line}"
            yield test_case, make_test_id(filename, path)
```

<img width="1115" alt="screen shot 2019-02-16 at 5 36 58 pm" src="https://user-images.githubusercontent.com/3391420/52897589-9492eb00-3211-11e9-938f-4001e112ae9d.png">

Reply:
That's amazing! Added it, can you check that the filename and line format is recognized by vscode (my editor/terminal doesn't support this unfortunately)?

Reply:
Can confirm the update works on my vscode.
Vim kind of supports it too http://vim.wikia.com/wiki/Open_file_under_cursor